draft-ietf-jsonpath-base.md

```diff
@@ -1111,8 +1110,12 @@ filter selector iterates over the array or object.
 ##### Existence Tests
 {: unnumbered}
 
-A singular path by itself in a Boolean context is an existence test which yields true if the path selects a node and yields false if the path does not select a node.
-This existence test — as an exception to the general rule — also works with nodes with structured values.
+A path by itself in a Boolean context is an existence test which yields true if the path selects at least one node and yields false if the path does not select a node.
```

Comment:
maybe "... does not select any nodes"?

Reply:
I'm equally comfortable with either phrase, so I changed to use the suggestion.